SimG4CMS/Calo/plugins/EcalSimHitStudy.cc

```diff
@@ -40,7 +41,7 @@
 #include <string>
 #include <vector>
 
-//#define EDM_ML_DEBUG
+#define EDM_ML_DEBUG
```

Comment:
@bsunanda , you left DEBUG for purpose or forget?

Reply:
I left it on purpose. I am working on a PR after this where I need to change this class as well. That will provide a true comparison between ddd and dd4hep. Could you sign this PR?